content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md

```diff
@@ -0,0 +1,69 @@
+---
+title: experimental.integral() function
+description: >
+  The `integral()` function computes the area under the curve per unit of time of subsequent non-null records.
+  Input tables must have `_time` and `_value` columns.
+menu:
+  influxdb_2_0_ref:
+    name: experimental.integral
+    parent: Experimental
+weight: 302
+related:
+  - /{{< latest "influxdb" "v1" >}}/query_language/functions/#integral, InfluxQL – INTEGRAL()
+  - /influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/integral/
+introduced: 0.106.0
+---
+
+The `experimental.integral()` function computes the area under the curve per
+[`unit`](#unit) of time of subsequent non-null records.
+The curve is defined using `_time` as the domain and record values as the range.
+**Input tables must have `_time` and `_value` columns.**
+
+```js
+integral(
+  unit: 10s,
+  interpolation: ""
+)
+```
+
+## Parameters
+
+### unit
+Time duration used when computing the integral.
+
+_**Data type:** Duration_
+
+### interpolate
```

Comment:
Is interpolate or interpolation the parameter?

Reply:
It's `interpolate`. I updated the code block above. Nice catch!